build.gradle

```diff
@@ -127,10 +133,13 @@ dependencies {
     )
 
     testImplementation(
-            [group: 'junit', name: 'junit', version: '4.12'],
+            [group: 'junit', name: 'junit', version: '4.13'],
             [group: 'com.sparkjava', name: 'spark-core', version: '2.2'],
             [group: 'org.glassfish.jersey.core', name: 'jersey-common', version: '2.22.4']
     )
+    testRuntimeOnly(
+           [group: 'org.junit.vintage', name:'junit-vintage-engine', version:'5.8.2']
```

Comment:
This test runtime dependency is the engine that runs junit4 tests since gradle seems to assume you're using 5 now.